_runners/vault.py

```diff
@@ -73,7 +73,10 @@ def _get_policies(minion_id, config):
 
   policies = []
   for pattern in policyPatterns:
-    policies.append(pattern.format(**mappings))
+    try:
+      policies.append(pattern.format(**mappings))
+    except KeyError:
```

Comment:
Hm, I don't think so? The error indicates that your string is malformed: `ValueError: Missing ']' in format string`